.travis.yml

```diff
@@ -26,10 +26,12 @@ matrix:
     - os: osx
       r: release
       if: branch = master
+      brew_packages: libgit2
```

Comment:
Is travis having problems?

Reply:
Yes, only on mac. says it needs that package for git2r

Reply:
It's happening on all estimatr branches

Reply:
:( 

I would cherry pick this edit (git cherrypick --help), push that (and only that) on to master, then in the other PRs do the update from master button

Reply:
Got it.